### PR TITLE
HD CSS Color Guide

### LINKED ISSUE #4525: [A11y][CSS] High Definition CSS Color Guide [1/10/2023]
* Proposed title: High Definition CSS Color Guide
* Proposed by: argyle
* Related areas: Accessibility,CSS
* Content type: Reference (https://documentation.divio.com/reference/)
* Deadline: 1/10/2023
* Deadline explanation: Chrome 109 release, which will have a majority of these color features hitting stable
* [Draft](https://docs.google.com/document/d/1pMnudZk0NYDKiXnd81Mr_wq5td_EyBge2MYGfZGArDQ/edit)
* Content summary: CSS Color 4 and 5 brings more color spaces, color manipulation functions and overall more colors and gradients to choose from, and this post intends to serve as a canonical guide to the syntax, features, and capabilities of these CSS specs.

### PR DESCRIPTION
**Pull request by argyleink**
Fixes #4525

<br>

Changes proposed in this pull request:
- New article: HD Color Guide
- New blog post to introduce the article: Meet The New CSS Color Spaces
- Update to `color-mix()` post: links to the new HD Color Guide

<br>

To Do:
- [x] get final reviews from DCC editors
- [x] get final reviews from engineers
- [x] update blog post hero image and subtitle
- [x] update dates

### CODE REVIEW

**Comment by argyleink** [comment]
missing gradient hue interpolation CSS examples around shorter, longer etc. only shows the colorjs versions.

**Comment by rachelandrew** [comment]
@argyleink If this is shipping in 110 we need someone to update Chrome Status https://chromestatus.com/feature/6280020765966336

**Comment by argyleink** [comment]
> @argyleink If this is shipping in 110 we need someone to update Chrome Status  

i'm still waiting on confirmation after holidays, will pass on this tip though and report back when I know

**Review by ajul — COMMENTED** [review]
Left some comments. Much is subjective, feel free to take or ignore anything.

File: site/en/blog/high-definition-css-color-guide/index.md
Comment (argyleink) [reply site/en/blog/high-definition-css-color-guide/index.md:21]: coo, will update

File: site/en/blog/high-definition-css-color-guide/index.md
Comment (argyleink) [reply site/en/blog/high-definition-css-color-guide/index.md:35]: sgtm, updating

File: site/en/blog/high-definition-css-color-guide/index.md
Comment (argyleink) [reply site/en/blog/high-definition-css-color-guide/index.md:65]: done.

File: site/en/blog/high-definition-css-color-guide/index.md
Comment (argyleink) [reply site/en/blog/high-definition-css-color-guide/index.md:56]: done.

File: site/en/blog/high-definition-css-color-guide/index.md
Comment (argyleink) [reply site/en/blog/high-definition-css-color-guide/index.md:125]: done.

File: site/en/blog/high-definition-css-color-guide/index.md
Comment (argyleink) [reply site/en/blog/high-definition-css-color-guide/index.md:219]: done. will have both

File: site/en/blog/high-definition-css-color-guide/index.md
Comment (argyleink) [reply site/en/blog/high-definition-css-color-guide/index.md:329]: thx, done.

File: site/en/blog/high-definition-css-color-guide/index.md
Comment (argyleink) [reply site/en/blog/high-definition-css-color-guide/index.md:1001]: hm yes, i dont really talk about ints vs floats at all. lemme think on that thanks!

File: site/en/blog/high-definition-css-color-guide/index.md
Comment (argyleink) [reply site/en/blog/high-definition-css-color-guide/index.md:1311]: yeah, what's the issue / suggestion?

Comment (ajul) [reply site/en/blog/high-definition-css-color-guide/index.md:1311]: The "p".is missing.

Comment (argyleink) [reply site/en/blog/high-definition-css-color-guide/index.md:1311]: ahh! nice =)
viva peer reviews!

Comment (argyleink) [reply site/en/blog/high-definition-css-color-guide/index.md:1311]:
```suggestion
color: color(display-p3 1 0 0);
```

File: site/en/blog/high-definition-css-color-guide/index.md
Comment (argyleink) [reply site/en/blog/high-definition-css-color-guide/index.md:1318]:
```suggestion
color: color(display-p3 1 0 0);
```

**Comment by argyleink** [comment]
Chrome 110 confirmed. Chrome status update request made.

**Comment by argyleink** [comment]
@rachelandrew here's the [Chrome Status their working with](https://chromestatus.com/feature/5147752829288448). they're going to merge / delete one so there's less confusion.

**Comment by rachelandrew** [comment]
@argyleink there is a separate one for color-mix() as well... https://chromestatus.com/feature/5659115386830848 these probably need disentangling.

File: site/en/blog/high-definition-css-color-guide/index.md
Comment (argyleink) [inline site/en/blog/high-definition-css-color-guide/index.md:1467]:
```suggestion
  --neon-red: rgb(100% 0 0);
  --neon-blue: rgb(0 0 100%);
```

**Review by mysteryDate — COMMENTED** [review]
A good start! I have many thoughts. I'll let you get to these before I move on to the rest of the document.

**Comment by rachelandrew** [comment]
@argyleink it looks like this should have had more review in the doc? Can we take it back there until people are happy with it and you've got all your LGTMs?

File: site/en/blog/high-definition-css-color-guide/index.md
Comment (argyleink) [reply site/en/blog/high-definition-css-color-guide/index.md:92]: fixed in doc

File: site/en/blog/high-definition-css-color-guide/index.md
Comment (argyleink) [reply site/en/blog/high-definition-css-color-guide/index.md:120]: moved to doc

File: site/en/blog/high-definition-css-color-guide/index.md
Comment (argyleink) [reply site/en/blog/high-definition-css-color-guide/index.md:125]: moved to doc

File: site/en/blog/high-definition-css-color-guide/index.md
Comment (argyleink) [reply site/en/blog/high-definition-css-color-guide/index.md:129]: moved to doc 👍🏻

File: site/en/blog/high-definition-css-color-guide/index.md
Comment (argyleink) [reply site/en/blog/high-definition-css-color-guide/index.md:140]: moved to doc

File: site/en/blog/high-definition-css-color-guide/index.md
Comment (argyleink) [reply site/en/blog/high-definition-css-color-guide/index.md:145]: moved to doc

File: site/en/blog/high-definition-css-color-guide/index.md
Comment (argyleink) [reply site/en/blog/high-definition-css-color-guide/index.md:148]: moved to doc

File: site/en/blog/high-definition-css-color-guide/index.md
Comment (argyleink) [reply site/en/blog/high-definition-css-color-guide/index.md:170]: moved to doc

File: site/en/blog/high-definition-css-color-guide/index.md
Comment (argyleink) [reply site/en/blog/high-definition-css-color-guide/index.md:235]: moved to doc

File: site/en/blog/high-definition-css-color-guide/index.md
Comment (argyleink) [reply site/en/blog/high-definition-css-color-guide/index.md:246]: moved to doc

File: site/en/blog/high-definition-css-color-guide/index.md
Comment (argyleink) [reply site/en/blog/high-definition-css-color-guide/index.md:250]: fixed in doc

File: site/en/blog/high-definition-css-color-guide/index.md
Comment (argyleink) [reply site/en/blog/high-definition-css-color-guide/index.md:251]: moved to doc

File: site/en/blog/high-definition-css-color-guide/index.md
Comment (argyleink) [reply site/en/blog/high-definition-css-color-guide/index.md:261]: moved to doc

File: site/en/blog/high-definition-css-color-guide/index.md
Comment (argyleink) [reply site/en/blog/high-definition-css-color-guide/index.md:267]: fixed in doc

File: site/en/blog/high-definition-css-color-guide/index.md
Comment (argyleink) [reply site/en/blog/high-definition-css-color-guide/index.md:282]: moved to doc

File: site/en/blog/high-definition-css-color-guide/index.md
Comment (argyleink) [reply site/en/blog/high-definition-css-color-guide/index.md:374]: moved to doc

File: site/en/blog/high-definition-css-color-guide/index.md
Comment (argyleink) [reply site/en/blog/high-definition-css-color-guide/index.md:380]: moved to doc

File: site/en/blog/high-definition-css-color-guide/index.md
Comment (argyleink) [reply site/en/blog/high-definition-css-color-guide/index.md:390]: moved to doc

File: site/en/blog/high-definition-css-color-guide/index.md
Comment (argyleink) [reply site/en/blog/high-definition-css-color-guide/index.md:395]: moved to doc

File: site/en/blog/high-definition-css-color-guide/index.md
Comment (argyleink) [reply site/en/blog/high-definition-css-color-guide/index.md:405]: moved to doc

File: site/en/blog/high-definition-css-color-guide/index.md
Comment (argyleink) [reply site/en/blog/high-definition-css-color-guide/index.md:413]: moved to doc

File: site/en/blog/high-definition-css-color-guide/index.md
Comment (argyleink) [reply site/en/blog/high-definition-css-color-guide/index.md:451]: moved to doc

File: site/en/blog/high-definition-css-color-guide/index.md
Comment (argyleink) [reply site/en/blog/high-definition-css-color-guide/index.md:462]: moved to doc

File: site/en/blog/high-definition-css-color-guide/index.md
Comment (argyleink) [reply site/en/blog/high-definition-css-color-guide/index.md:502]: totally, will def make a gradient comparison 👍🏻

Comment (argyleink) [reply site/en/blog/high-definition-css-color-guide/index.md:502]: moved to doc

File: site/en/blog/high-definition-css-color-guide/index.md
Comment (argyleink) [reply site/en/blog/high-definition-css-color-guide/index.md:508]: moved to doc

File: site/en/blog/high-definition-css-color-guide/index.md
Comment (argyleink) [reply site/en/blog/high-definition-css-color-guide/index.md:515]: yep def! thx!

File: site/en/blog/high-definition-css-color-guide/index.md
Comment (argyleink) [reply site/en/blog/high-definition-css-color-guide/index.md:1001]: moved to doc

File: site/en/blog/high-definition-css-color-guide/index.md
Comment (argyleink) [reply site/en/blog/high-definition-css-color-guide/index.md:90]: moved to the doc

**Review by argyleink — COMMENTED** [review]
All comments moved to the Google doc 🙂 Thanks everyone.

PR will receive an update once the document is approved again 👍🏻

File: site/en/blog/meet-the-new-css-color-spaces/index.md
Comment (rachelandrew) [inline site/en/blog/meet-the-new-css-color-spaces/index.md:15]: I think you are using a different image for the blog

File: site/en/blog/meet-the-new-css-color-spaces/index.md
Comment (rachelandrew) [inline site/en/blog/meet-the-new-css-color-spaces/index.md:21]:
```suggestion
CSS color 4 brings a large set of tools and features to CSS for managing and
handling color. I've written the [High Definition Color
Guide](/articles/high-definition-css-color-guide/) to cover all these new
features. In this guide you will learn:
```

File: site/en/blog/meet-the-new-css-color-spaces/index.md
Comment (rachelandrew) [inline site/en/blog/meet-the-new-css-color-spaces/index.md:36]:
```suggestion
- What is a color gamut?
- Human visual gamut.
- What is a color space?
- How to access more colors, new spaces, and debug results.
- A review of the classic color spaces.
- Meet the new web color spaces.
- Color interpolation.
- Gamut clamping.
- Choosing a color space.
- Migrating to HD CSS color.
- Checking for gamut and color space support.
- Debugging color with Chrome DevTools.

I hope this finds you well and answers all your color questions. 🙂
```

File: site/en/articles/high-definition-css-color-guide/index.md
Comment (rachelandrew) [inline site/en/articles/high-definition-css-color-guide/index.md:192]: Add alt text

**Review by rachelandrew — APPROVED** [review]
Approved so as not to be blocking, but there's a couple of alt text nits along with a couple of edits. Please check those before merging.

File: site/en/blog/meet-the-new-css-color-spaces/index.md
Comment (argyleink) [reply site/en/blog/meet-the-new-css-color-spaces/index.md:15]: got it thx!

**Pull request merged**
